how-tos/releaseduty_overview.md

```diff
@@ -0,0 +1,21 @@
+# ReleaseDuty Statement of Work / Overview
+
+ReleaseDuty is a duty cycle where an engineer is responsible for operational
+tasks related to releasing new versions of Firefox.  Two people are assigned to
+ReleaseDuty every six weeks, preferably in different timezones to extend
+available coverage.  There are specific days in a cycle when set tasks must be
+done, such as creating a beta or release candidate, and other unscheduled work
+that depends on issues that need fixing, and special requirements that specific
+release might have.
+
+ReleaseDuty is responsible for:
+* Performing operational tasks for incoming Nightly, DevEdition, Beta, Release Candidate and final releases
+* Merging repositories to create new beta, release candidates, releases and extended support releases.
+* Coordinating with other teams, such as Release Management, Quality Engineering, Sheriffs, and other developers
+* Fixing and improving the release automation, including both tools and processes
+* Maintaining a logbook of issues and progress using releasewarrior.
+* Arranging event post-mortems, if required.
+
+The individual accountable for a release is known as the 'release owner'.
```

Comment:
`release_owner` in this context more or less refers to RelMan. Might be worth mentioning that here.

Reply:
We should mention it's a specific member of release management, as I don't think it's a group

Reply:
Precisely. Sorry, that's what I meant actually, slipping a "Release Management release owner" there to make it clear it's not RelEng.